service/controller/resource/heartbeat/create.go

```diff
@@ -68,5 +68,12 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		r.logger.LogCtx(ctx, "level", "debug", "message", "heartbeat is up to date")
 	}
 
+	// Trigger initial ping to ensure opsgenie alerts when an issue arise.
```

Comment:
This comment doesn't follow to me, or feels more like 'this is what all the code does'

Language more towards 'the initial ping to the heartbeat is there to move the heartbeat from inactive to active' would make sense to me

Reply:
:+1: